lib/event_consumer.rb

```diff
@@ -0,0 +1,103 @@
+# The EventConsumer currently only listens on the aker.events.catalogue queue
+class EventConsumer
+
+  def initialize(config = {})
+    @broker_host = config[:broker_host]
+    @broker_port = config[:broker_port]
+    @broker_vhost = config[:broker_vhost]
+    @broker_username = config[:broker_username]
+    @broker_password = config[:broker_password]
+    @exchange_name = config[:exchange_name]
+    @catalogue_queue_name = config[:catalogue_queue_name]
+  end
```

Comment:
It's generally better to use `fetch` for getting values out of Hashes. It'll raise an exception rather than returning `nil` for a missing key.